track_digital/track/templates/en/layout-en.html

```diff
@@ -39,19 +39,22 @@ <h2 class="sr-only">Language selection</h2>
 
 	     <nav class="bg-grey-light">
 			<div class="container mx-auto flex">
-				<div class="{{ self.pageid() | site_title(false) }}">
-					<h2 class="py-6 text-lg sm:text-xl hidden sm:block"><a class="underline text-black" href="/en/organizations/">Track digital security compliance</a></h2>
+				<div class="{{ self.pageid_fr() | site_title(false) }}">
```

Comment:
This filter only cares about whether or not the value of the pageid is "index", which is the same in french and english, so the language passed here is irrelevant. Kept it _fr and _en consistent with the rest of the layout